gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
```

Comment:
This comment concerning the class docstring Sphinx formatting isn't addressed yet.